src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.bigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+
+}
+
+.search-bar {
+  position: relative;
+  width: 100%;
+}
+
+.icon {
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.search {
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  font-weight: 400;
+}
+
+.search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-sizing: border-box;
+  font-weight: 900;
+}
```

Comment:
don't duplicate styles which you already have in `.search` class. also fix for `.icon, .search--small`

Reply:
ok